FeedMe/FeedMe/DayMenuViewController.swift

```diff
@@ -56,4 +57,16 @@ class DayMenuViewController: UIViewController,  UITableViewDelegate, UITableView
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        
+        let storyboardName  = Bundle.main.object(forInfoDictionaryKey: "UIMainStoryboardFile")
+        let storyBoard = UIStoryboard(name: storyboardName as! String, bundle: Bundle.main)
```

Comment:
Instead of instantiating a new storyboard wouldn't it be better to use the storyboard property of the controller? 
https://developer.apple.com/reference/uikit/uiviewcontroller/1621399-storyboard